cv/qc-cv.obo

```diff
@@ -190,2207 +191,270 @@ name: MS metric
-relationship: has_relation MS:4000013 ! single run based
-relationship: has_units UO:0000189 ! count unit
 synonym: "MS2-Count" EXACT []
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000013 ! single run based
+relationship: has_metric_category MS:4000010 ! ID free
+relationship: has_metric_category MS:4000024 ! MS2 metric
+relationship: has_value_type xsd\:int ! The allowed value-type for this CV term
+relationship: has_units UO:0000189 ! count unit
 
 [Term]
 id: MS:4000062
 name: MS2 density per quantile
 def: "The first to n-th quantile of MS2 scan peak counts." [PSI:QC]
-is_a: MS:4000004 ! n-tuple
-is_a: MS:4000010 ! ID free
-is_a: MS:4000024 ! MS2 metric
-relationship: has_relation MS:1000035 ! peak picking
-relationship: has_relation MS:4000013 ! single run based
+comment: the name implies the density of something in quantiles of another thing (so quarters would be more appropriate), but I think the definition intention is to divide the range of MS" densities and report the quantiles?
 synonym: "MS2-Density-Q1" RELATED []
 synonym: "MS2-Density-Q2" RELATED []
 synonym: "MS2-Density-Q3" RELATED []
+relationship: has_metric_category MS:4000013 ! single run based
+relationship: has_metric_category MS:4000010 ! ID free
+relationship: has_metric_category MS:4000024 ! MS2 metric
+relationship: has_value_type xsd\:int ! The allowed value-type for this CV term
+relationship: has_value_concept NCIT:C45781 ! Density
+relationship: has_units UO:0000189 ! count unit
 
 [Term]
 id: MS:4000063
 name: MS2 known precursor charges fractions
-def: "The fraction of MS/MS precursors of the corresponding charge. The fractions [0,1] are given in the 'Fraction' column, corresponding charges in the 'Charge state' column. The highest charge state is to be interpreted as that charge state or higher. " [PSI:QC]
-is_a: MS:4000006 ! table
-is_a: MS:4000010 ! ID free
-is_a: MS:4000024 ! MS2 metric
-is_a: MS:4000025 ! ion source metric
-relationship: has_relation MS:1000041 ! charge state
-relationship: has_relation MS:4000013 ! single run based
-relationship: has_column: MS:4000238 ! Charge state
-relationship: has_column: MS:4000239 ! Fraction
+def: "The fraction of MS/MS precursors of the corresponding charge. The fractions [0,1] are given in the 'Fraction' column, corresponding charges in the 'Charge state' column. The highest charge state is to be interpreted as that charge state or higher." [PSI:QC]
 synonym: "MS2-PrecZ-1" RELATED []
 synonym: "MS2-PrecZ-2" RELATED []
 synonym: "MS2-PrecZ-3" RELATED []
 synonym: "MS2-PrecZ-4" RELATED []
 synonym: "MS2-PrecZ-5" RELATED []
 synonym: "MS2-PrecZ-more" RELATED []
+is_a: MS:4000006 ! table
+relationship: has_metric_category MS:4000013 ! single run based
+relationship: has_metric_category MS:4000010 ! ID free
+relationship: has_metric_category MS:4000024 ! MS2 metric
+relationship: has_metric_category MS:4000025 ! ion source metric
+relationship: has_column: MS:4000238 ! Charge state column
```

Comment:
Based on some earlier feedback from the PSI-MS-CV folks, they were reluctant to have to redefine terms used for columns as new terms, but rather suggested just using existing terms directly as columns. This prohibits full validation of column types, but doesn't have any major downsides as far as I understand? Maybe not a bad thing for simplicity of the CV and to avoid near-duplication? See Eric's comment in #164.